### PR TITLE
Bluetooth Fast Pair: Battery Notification extension Kconfig

### PR DESCRIPTION
**Pull request by kapi-no**

### CODE REVIEW

File: include/bluetooth/adv_prov/fast_pair.h [hunk 1]
```diff
@@ -45,6 +45,9 @@ void bt_le_adv_prov_fast_pair_enable(bool enable);
 void bt_le_adv_prov_fast_pair_show_ui_pairing(bool enable);
 
 /** Set advertising battery mode in Fast Pair not dicoverable advertising.
+ *
+ * This API must not be used when the @kconfig{CONFIG_BT_FAST_PAIR_BN} Kconfig option is enabled.
```
[the remark this answers is not in the document]
Comment (mkapala-nordic) [reply include/bluetooth/adv_prov/fast_pair.h:49]: Or invert `This API must not be used when` -> `This API can be used only when`

**Review by mkapala-nordic — APPROVED** [review]
overall lgtm

File: include/bluetooth/adv_prov/fast_pair.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (kapi-no) [reply include/bluetooth/adv_prov/fast_pair.h:49]: Thanks, I decided to invert the first part of the statement to avoid double negations :)

**Comment by kapi-no** [comment]
The last force-push is a rebase only operation.

**Review by mkapala-nordic — APPROVED** [review]

**Review by alstrzebonski — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -561,6 +561,8 @@ Bluetooth libraries and services
 
   * Added:
 
+    * The :kconfig:option:`CONFIG_BT_FAST_PAIR_BN` Kconfig option allowing the user to enable the support for the Battery Notification extension.
+      Now, it is necessary to enable this new Kconfig option before accessing Fast Pair API elements that are associated with the Battery Notification extension.
```
Comment (annwoj) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:565]:
```suggestion
    * The :kconfig:option:`CONFIG_BT_FAST_PAIR_BN` Kconfig option that enables support for the Battery Notification extension.
      You must enable this option to access Fast Pair API elements associated with the Battery Notification extension.
```

**Review by annwoj — COMMENTED** [review]
just a small suggestion

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst (hunk 2 above)
Comment (kapi-no) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst]: thanks, fixed

**Review by annwoj — APPROVED** [review]

**Pull request merged**